.infra/Pulumi.yaml

```diff
@@ -1,3 +1,6 @@
 name: api
-runtime: nodejs
+runtime:
+  name: nodejs
+  options:
+    packagemanager: pnpm
```

Comment:
wait with this change

Reply:
was gonna tag you about it

Reply:
reverted in 56dc5b41cb49860b087456d9faf358d695edef3d